spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurationsSenderConfigurationTests.java

```diff
@@ -74,17 +57,6 @@ void shouldBackOffOnCustomBeans() {
 		});
 	}
 
-	@Test
-	void shouldUseCustomHttpEndpointSupplierFactory() {
```

Comment:
Instead of removing this test, it should be adjusted. The `HttpEndpointSupplier` is also used by the `ZipkinHttpClientSender`.

1. Remove the `FilteredClassLoader`.
2. Retrieve the `ZipkinHttpClientSender` from the `context`.
3. Verify that `ZipkinHttpClientSender` is configured  to use `CustomHttpEndpointSupplier`

Reply:
I agree.

Reply:
Implemented